src/engine/bufferscalers/enginebufferscalerubberband.h

```diff
@@ -45,4 +50,6 @@ class EngineBufferScaleRubberBand : public EngineBufferScale {
 
     // Holds the playback direction
     bool m_bBackwards;
+
+    bool m_useEngineV3;
```

Comment:
The term engine V3 becomes weak, if they introduce more options in the lifetime of V3
```suggestion
    bool m_useEngineFiner;
```